install/Create_System_Configuration_Using_Cluster_Discovery_Service.md

```diff
@@ -0,0 +1,397 @@
+# Create System Configuration Using Cluster Discovery Service
+
+This stage walks the user through creating the configuration payload for the system using Cluster Discovery Service (CDS) to generate seed files and paddle file.
+
+Run the following steps before starting any of the system configuration procedures.
+
+1. (`pit#`) Make the `prep` and `cds` directory.
+
+   ```bash
+   mkdir -pv "${PITDATA}/prep/cds"
+   ```
+
+1. (`pit#`) Change into the `cds` directory.
+
+   ```bash
+   cd "${PITDATA}/prep/cds"
+   ```
+
+1. (`pit#`) Set `CVT_PATH` variable.
+
+   ```bash
+   CVT_PATH=/opt/src/cluster-config-verification-tool/
+   ```
+
+## Topics
+
```

Comment:
Is the user expected to follow these in order, as a procedure? The use of a numbered list implies that. But earlier on the page it says "Run the following steps before starting any of the system configuration procedures", which makes it sound like maybe the user would only be doing some of these.

If this is one procedure that is intended to be done in order, then the section above should just be added as a prerequisites section to the steps below.

Reply:
The added procedure was almost similar to the earlier procedure that was using the SHCD method. So the similar statements were used. This also can be added as a separate Prerequisite section before the actual procedures. 

To maintain the similar flow of the earlier content, I added it under the same statement. 

Please suggest if this needs to be changed.